### ISSUE TITLE
Google sign-in error with webview

### ISSUE THREAD

**Issue by tiancaipipi110** [gree/unity-webview#384]
Hello, I'm accessing my website's login in form's Google & Facebook sign-in. However Google sign-in is throwing me an error (facebook login works fine): 
`403 Error: disallowed_useragent. Google can't sign you in safely inside this app. You can use Google sign-in by visiting this app's website in a browser like Safari or Chrome.`
I'm wondering if there's a way to work around this?

**Comment by juscelinotanaka** [comment]
I know that it is related with some security issues from the Google Sign In API. It is possible to be fixed if you can set the user agent to anyone like the Android or iOS browsers (I.e. ‘Mozilla/5.0 (platform; rv:geckoversion) Gecko/geckotrail Firefox/firefoxversion’) but I am not sure if we have the ability to set the user agent in this plugin. If we do so then try setting up it and you will be able to workaround this issue.

**Comment by tiancaipipi110** [comment]
@juscelinotanaka Yes, that's what I though. But what do you mean that we might not have the ability to set user agent in this plugin? Is it a technical issue?

**Comment by KojiNakamaru** [comment]
You can specify your custom user agent (cf. https://github.com/gree/unity-webview/blob/f35310531ce6598dba17459d8384eaa36bec2dbc/plugins/WebViewObject.cs#L331 ), though I'm not sure it works for Google sign-in API and basically we should avoid to utilize webview for it (cf. https://github.com/gree/unity-webview/issues/313#issuecomment-381836584 ).
